packages/protocol/contracts/common/interfaces/IScoreManager.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.5.13 <0.9.0;
+
+interface IScoreManager {
```

Comment:
as said in some other PR. A Score manager has getters for scores and only that. Setters are an implemetation detail, you can create another socre manager that computes scores some other way and setters wouldn't make sense there.

Instead just have some IScoreManager or IScoreREader, and that's the dependency EpochManager needs

Reply:
in this case maybe move to another interface `IScoreSetter` or maybe for now not needed at all.

Reply:
resolving this, as epochManager already uses `IScoreReader`